integration_tests/src/main/python/conftest.py

```diff
@@ -86,6 +87,16 @@ def is_utc():
 def is_not_utc():
     return not is_utc()
 
+def is_supported_time_zone():
+    """
+    Is current TZ supported, forward to Java TimeZoneDB to check
+    """
+    tz = get_test_tz()
+    jvm = spark_jvm()
+    ret = jvm.org.apache.spark.sql.rapids.TimeZoneDB.isSupportedTimeZone(tz)
+    print("my debug: is_supported_time_zone " + str(ret))
```

Comment:
if this print is useful it should not be "my debug"

Reply:
Done, remove the debug print.